### PR TITLE
merge dev to master

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -142,9 +142,11 @@
     "@material-ui/lab": "^4.0.0-alpha.60",
     "apexcharts": "3.32.1",
     "i18next-http-backend": "^1.4.1",
+    "market-sdk": "^2.1.0",
```
Comment (FriskyHamTitz) [inline package.json:145]: Why is market XYZ here

File: src/App.tsx [hunk 2]
```diff
@@ -108,90 +111,104 @@ function Gelato({ children }: { children?: React.ReactNode }) {
...
+                        <PageLayout>
+                          <SwapPage />
+                        </PageLayout>
+                      </Route>
+                      {/* <Route exact path='/lend'>
+                        <PageLayout>
+                          <LendPage />
+                        </PageLayout>
+                      </Route> */}
+                      <Route exact path='/lend/detail'>
```
Comment (FriskyHamTitz) [inline src/App.tsx:145]: Remove lend details

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]
TBH we should only be merging the ads feature branch to master, but if you wanna do it this way hid all the market XYZ routers

File: package.json (hunk 1 above)
Comment (totop716) [reply package.json:145]: I think we don't need to remove the market sdk package.

**Review by FriskyHamTitz — APPROVED** [review]

**Review by sameepsi — APPROVED** [review]

**Pull request merged**